### PR TITLE
[SDK] Fix Darwin CLI build, add uninstall instructions to build.sh footer

### PR DESCRIPTION
**Pull request by nickbp**
- Darwin binary must not be UPX-encoded, or else it immediately produces `Killed: 9`
- Include all steps to reinstall a service in build output

### CODE REVIEW

File: tools/publish_aws.py [hunk 1]
```diff
@@ -152,9 +152,6 @@ def upload(self):
         # print universe url early
         universe_url = self._upload_artifact(universe_path)
         logger.info('---')
-        logger.info('Built and uploaded stub universe:')
-        logger.info(universe_url)
-        logger.info('---')
```
Comment (jsrodman) [inline tools/publish_aws.py]: Unrelated?

Comment (nickbp) [inline tools/publish_aws.py]: Yes, it's redundant given the install steps provided at the end
